gtest-parallel

```diff
@@ -181,9 +188,16 @@ class FilterFormat:
     stdout_lock.release()
 
   def end(self):
-    self.print_tests("FAILED TESTS", self.failures)
-    interruptions = set(self.started) - set(self.finished)
-    self.print_tests("INTERRUPTED TESTS", interruptions)
+    if self.failures:
+      self.print_tests("FAILED TESTS", self.failures)
+      self.write_logs_to_file('Failed tests.log', self.failures)
```

Comment:
Let's not use spaces in the filename, that's never a good idea. I'd also prefer all lowercase.